<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectDialog</class>
 <widget class="QDialog" name="ConnectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1004</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>310</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="inputLabel_1">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>50</y>
     <width>400</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Input Device 1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputEdit_1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="inputLabel_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>90</y>
     <width>400</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Input Device 2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputEdit_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="inputLabel_3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>130</y>
     <width>400</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Input Device 3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputEdit_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="inputLabel_4">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>170</y>
     <width>400</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Input Device 4</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputEdit_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="inputLabel_5">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>210</y>
     <width>400</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Input Device 5</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputEdit_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="inputLabel_6">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>250</y>
     <width>400</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Input Device 6</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputEdit_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
